api/src/service/domain/workflow/workflowitem_create.ts

```diff
@@ -171,12 +171,7 @@ export async function createWorkflowitem(
   const documentUploadedEvents: BusinessEvent[] = [];
 
   if (reqData.documents?.length) {
-    const documentsCount = reqData.documents.filter((d) => d.base64).length;
```

Comment:
@SamuelPull is there a reason you removed this lines? This was enabling a deployment without a storage service while still enable to use external links.

Reply:
no, git removed these when merging branches.